Edabit/Medium/Remove the Letters ABC.js

```diff
@@ -0,0 +1,6 @@
+//Problem link: https://edabit.com/challenge/RftQjZs3srJnRBEH8
```

Comment:
add space wherever applicable
```suggestion
// Problem link: https://edabit.com/challenge/RftQjZs3srJnRBEH8
```